demos/projects/ESPRESSIF/az-ca-recovery/README.md

```diff
@@ -243,6 +251,16 @@ openssl x509 -in recovery-public-cert.pem -modulus -noout | sed s/Modulus=// | s
 openssl x509 -in recovery-public-cert.pem --text -noout | grep Exponent | sed -r 's/.*Exponent: .*\((.*)\)/\1/g'
 ```
 
+For Powershell:
+
+```powershell
+# Modulus
+openssl x509 -in recovery-public-cert.pem -modulus -noout | ForEach-Object { $_ -replace 'Modulus=', '' } | ForEach-Object { $_ -replace '(..)', '0x$1, ' }
+
+# Exponent
+openssl x509 -in recovery-public-cert.pem --text -noout | Select-String Exponent |  ForEach-Object { $_ -replace 'Exponent: .*\((.*)\)', '$1' }
```

Comment:
This did not work on my `openssl` version on Windows 11 PowerShell.  When I used a single hyphen `-text -noout` it did.  

When I did this on bash (WSLv2 Ubuntu based distro specfiically) it seemed perfectly happy with both a single hyphen and double hyphen.

Is there a reason we need the double hyphen version?  If not, recommend we move to single hyphen.  Potentially points to me having an older openssl on Win (which I'll check in parallel) but I'm sure other customers could be in same boat I am so ideally we can make 1 cmd to make everyone work.